### PR TITLE
Add more examples to 787be828-9a7f-4998-9bc6-a8b06f2d77de

### PR DESCRIPTION
**Pull request by wheresrhys**
Add more examples to clarify what improving delivery process can look like when the process is strongly owned by one or more non-engineers
Addresses #90

### CODE REVIEW

File: data/competencies.yml [hunk 1]
```diff
@@ -510,6 +510,8 @@
   summary: Improves delivery process and encourages others to do the same
   examples:
     - Updates the scrum process to fit the changing needs of the team
+    - Champions technical issues that effect delivery such as release cycles, dealing with tech debt and bug fixes
```
Comment (alicebartlett) [inline data/competencies.yml]: This should be "affect" not "effect"

Comment (alicebartlett) [reply data/competencies.yml]: ... Yes... I totally set you up for this because it was me that wrote the copy originally 🙈

Comment (wheresrhys) [reply data/competencies.yml]: 👋 🤜

Comment (wheresrhys) [reply data/competencies.yml]: (waving fist, not threatening violent retribution... although arguably waving a fist is that. Where's that retreat emoji when I need it)

**Comment by wheresrhys** [comment]
:trollface:

**Pull request merged**